examples/kmm-sample/androidApp/src/main/java/io/realm/example/kmmsample/androidApp/MainActivity.kt

```diff
@@ -14,6 +15,10 @@ fun greet(): String {
 }
 
 class MainActivity : AppCompatActivity() {
+
+    private lateinit var countTV: TextView
+    private var registration: Cancellable? = null
```

Comment:
why is this nullable? can't we use `var lateinit` instead?

Reply:
It is cleared when `cancel`-ing the subscription in `onPause`.

Reply:
No need to preemptively setting it to null, it will be GC'ed once the Activity goes out of scope. Plus the safe call on `onPause` is unnecessary since the lifecycle guarantees to call `onResume` before `onPause` anyway which ensure we set the `registration` var